test/backup.py

```diff
@@ -168,6 +184,9 @@ def test_backup(self):
     - bring up tablet_replica2 after the fact, let it restore the backup
     - check all data is right (before+after backup data)
     - list the backup, remove it
+
+    Args:
+      tablet_type: replica or rdonly.
```

Comment:
Should probably be "'replica' or 'rdonly'", i.e. values surrounded with quotes, otherwise it's not clear what these words mean.